Simplenote/Classes/SPAuthViewController.swift

```diff
@@ -243,23 +242,6 @@ private extension SPAuthViewController {
         navigationController?.navigationBar.applySimplenoteLightStyle()
     }
 
-    func setupAutolayoutConstraints() {
-        if #available(iOS 11, *) {
-            return
-        }
-
-        /// Nuke the current constraint: it's set against safeLayoutGuide, non existant in iOS 10
-        stackViewTopConstraint.isActive = false
-        stackView.removeConstraint(stackViewTopConstraint)
-
-        /// Establish a new top constraint, against the topLayoutGuide.
-        stackViewTopConstraint = stackView.topAnchor.constraint(equalTo: topLayoutGuide.bottomAnchor,
-                                                                constant: stackViewTopConstraint.constant)
-
-        /// And, please, forgive me for pushing this awful method.
-        stackViewTopConstraint.isActive = true
-    }
-
```

Comment:
curios, and the answer might belong above in the release notes, but I'm assuming that something in iOS 11 makes these constraints obsolete? are these constraints saved in a visual model somewhere? do they disappear because the underlying `UIViewController` changed its interface?

Reply:
Layout is set at the Interface Builder level (which is, then, translated into actual instances by NSCoder, when the `xib` is unarchived).

iOS 11 introduced a "Layout Guide" named [safeLayoutGuide](https://developer.apple.com/documentation/uikit/uiview/2891102-safearealayoutguide), which lets you define layout constraints for elements that must not be covered by navigation bars (or few ther components).

Since that layoutGuide was non-existant in iOS 10, we took a different approach there. Now, because we're bumping up the deployment target (minimum OS required), it's okay to remove that workaround-method.

Reply:
awesome. that would be really nice to read in the RELEASE-NOTES too…

```
 - iOS11 is new minimal to rely on `safeLayoutGuide` and remove prior custom constraints
```

I'm not trying to leverage my weight here; just coming in as an outsider sharing my thoughts. thanks for helping me understand!

Reply:
Huh I did reply on the RELEASE-NOTES comment, but somehow it didn't get actually posted!!

Thing is: the RELEASE-NOTES are picked by a script, localized, and posted to the AppStore. It should be as human friendly as possible!

Thanks for bringing it up though!!!